build.gradle

```diff
@@ -107,6 +127,31 @@ List<String> resolveIntelliJVersions(String versionKey) {
   return [versionValue]
 }
 
+String checkForNewEap (String intellijMajorVersion) {
+  Document htmlContent = Jsoup.connect(project.intellijSnapshotsUrl+"snapshots/").get()
+  Element link = htmlContent.select('a[href^='+project.intellijSnapshotsUrl+'snapshots/com/jetbrains/intellij/idea/ideaIC/'+intellijMajorVersion+'.][href$="-EAP-SNAPSHOT.pom"]')[0]
+  if (link) {
+    String attr = link.attr("href")
+    Pattern pattern = Pattern.compile("ideaIC-(.*?).pom")
+    Matcher matcher = pattern.matcher(attr)
+    matcher.find()
+    return matcher.group(1)
+  }
+}
+
+task updateEapOfLatestSupportedMajor {
+  Properties properties = intellijVersionsProperties()
+  File propFile = intellijVersionsFile()
+  String latestEap = properties.getProperty("eapOfLatestSupportedMajor")
+  char[] latestStable = properties.getProperty("latestStable").toCharArray()
```

Comment:
Would it be possible to rely on the already parsed values from `ext { ... }` in order not to re-parse the properties here? 🤔